src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -421,6 +421,22 @@ public async Task<IActionResult> SyncTriggers()
                 : StatusCode(StatusCodes.Status500InternalServerError, new { status = result.Error });
         }
 
+        [HttpGet]
+        [Route("admin/host/triggers")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
+        [ResourceContainsSecrets]
+        [RequiresRunningHost]
+        public async Task<IActionResult> GetTriggers()
+        {
+            _metricsLogger.LogEvent(MetricEventNames.GetTriggersInvoked);
+
+            var result = await _functionsSyncManager.GetTriggersAsync();
+
+            return result.Success
+                ? Ok(result)
+                : StatusCode(StatusCodes.Status403Forbidden, new { status = result.Error });
```

Comment:
Can we add a comment explaining the 403 forbidden. Suggestion:

``` CSharp
// GetTriggersAsync() does not swallow exceptions, so any failures will still be a 500. The only result.Success == false we need to consider is when the environment does not support sync triggers.
```

Reply:
Added the comment :)